services/auth/auth.go

```diff
@@ -121,7 +124,7 @@ func handleSignIn(resp http.ResponseWriter, req *http.Request, sess SessionStore
 	_ = sess.Delete("openid_determined_username")
 	_ = sess.Delete("twofaUid")
 	_ = sess.Delete("twofaRemember")
-	_ = sess.Delete("u2fChallenge")
+	_ = sess.Delete("webauthnAssertion")
```

Comment:
Do we still need to delete every keys in session? If yes, there is a `webauthnRegistration` (`registration`)

Reply:
Nope that's only set in settings.